AuthServ/AuthServer_Private.h

```diff
@@ -84,6 +83,7 @@ enum AuthServer_MsgIds
     e_AuthToCli_ScoreTransferPointsReply, e_AuthToCli_ScoreSetPointsReply,
     e_AuthToCli_ScoreGetRanksReply, e_AuthToCli_AcctExistsReply,
     e_AuthToCli_AgeReplyEx = 0x1000, e_AuthToCli_ScoreGetHighScoresReply,
+    e_AuthToCli_ServerCaps,
```

Comment:
Major :trollface: -- fortunately, the actual value of this enum is still correct (purely by coincidence)